app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,41 @@
+class RecipesController < ApplicationController
+  def index
+    @recipes = Recipe.all
```

Comment:
please add @user = current_user

Reply:
The `current_user` is unavailable since a user cannot log in. I will however do so when the log-in page is created. Thank you